src/views/AgmLocationChg.vue

```diff
@@ -78,45 +60,104 @@
               </h1>
             </header>
 
+            <!-- Help -->
             <ExpandableHelp helpLabel="Help with Annual General Meeting Extension">
               <template #content>
-                Help text, or sub-component, goes here.
+                <section class="agm-location-change-help">
+                  <h3 class="text-center">
+                    AGM Location Change Help
+                  </h3>
+                  <div class="mt-6">
+                    <p class="ml-1">
+                      This request for location change is only needed if the meetings are fully in-person. In general,
+                      these meetings are required to be held in British Columbia. However, there are exceptions to this
+                      rule. If the company's articles permit it or if the members agree, the meeting can be conducted at
+                      a location outside British Columbia. If the meeting combines both in-person and online
+                      participation, the location requirement only applies to the in-person component. If the meeting is
+                      entirely conducted online, these location restrictions do not apply.
+                    </p>
+                  </div>
+                </section>
```

Comment:
Looks good.

Did you consider making this an "aside" block instead of a section? Maybe ask Yui what he thinks.

Ref: https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
Are we going with Aside here, or Section? How did you decide?